opendebates/tests/test_submission.py

```diff
@@ -97,6 +99,38 @@ def test_post_submission(self):
         self.assertEqual(submission.approved, True)
         # vote count is kept on the submission model
         self.assertEqual(submission.votes, 1)
+        self.assertEqual(submission.current_votes, 1)
+        # but Vote objects are also created
+        votes = Vote.objects.filter(voter=self.voter)
+        self.assertEqual(1, len(votes))
+
+    def test_post_submission_after_previous_debate(self):
+        mode, _ = SiteMode.objects.get_or_create()
+        mode.previous_debate_time = timezone.make_aware(
+            datetime.datetime(2016, 1, 1, 12)
```

Comment:
Could this be `timezone.now() - datetime.timedelta(something)` instead?  I have a weird aversion to introducing a dependency on the code having being written after Jan 2016 (and being run on a machine whose clock isn't completely wrong).  Would also make the intention more explicit on a quick skim.